src/bindgen/config.rs

```diff
@@ -559,6 +559,9 @@ impl StructConfig {
 pub struct EnumConfig {
     /// The rename rule to apply to the name of enum variants
     pub rename_variants: RenameRule,
+    /// The rename rule to apply to the names of union body members.
+    /// Applied before rename_variants rename rule. Defaults to SnakeCase.
+    pub rename_union_body_members: RenameRule,
```

Comment:
So this is not for Rust unions (as in `union Foo {}` but C/C++ unions). I think this should just be `rename_fields` for consistency with `config.{struct,union}.rename_fields`, wdyt?

Reply:
Maybe `rename_variant_name_field` would be more explicit? I dunno.